### DISCUSSION TITLE
Apache Superset Embedded Not Working (Forbidden)

### DISCUSSION THREAD

**Discussion by butching** [apache/superset#23044] [General]
Im trying to embed a dashboard on my website and Im having this error
![image](https://user-images.githubusercontent.com/3095265/217730420-add12b6f-7fa9-4c92-8500-e3e620733aa8.png)

If you ask my why csrf is enabled false, its because it keeps on giving me this error
`{   "errors": [ { "message": "400 Bad Request: The CSRF session is missing.", "error_type": "GENERIC_BACKEND_ERROR", "level": "error", "extra": { "issue_codes": [ { "code": 1011, "message": "Issue 1011 - Superset encountered an unexpected error." } ] } }   ] }`

**Environment**

- browser type and version: Chrome/Version 109.0.5414.120
- superset version: superset docker: 2.0.1rc6
- other settings: `WTF_CSRF_ENABLED=False`
- any feature flags active: `"EMBEDDED_SUPERSET": True`

**Comment by butching** [comment]
I added GUEST_ROLE_NAME in the config and it works
reference here: https://github.com/apache/superset/issues/22258#issuecomment-1333387931

**Reply by laharidev1997** [reply]
i set up all the changes required for the request token to be generated. i am also passing my csrf token as a header in the postman but am still getting the same error.
<img width="1062" alt="Screenshot 2023-06-26 at 2 15 42 PM" src="https://github.com/apache/superset/assets/113213739/19bedf23-a988-4803-900e-c4eeaa4f4a4d">
